benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -193,12 +193,16 @@ private void ensureTopicsAreReady() throws IOException {
      */
     private void findMaximumSustainableRate(double currentRate) throws IOException {
         CountersStats stats = worker.getCountersStats();
+        PeriodStats periodStats = worker.getPeriodStats();
 
         int controlPeriodMillis = 3000;
         long lastControlTimestamp = System.nanoTime();
 
         RateController rateController = new RateController();
 
+        double targetP99EndToEndLatency = workload.p99EndToEndLatency;
```

Comment:
Can we pass these target latencies to the `RateController` constructor? (let's define them as member variables in RateController)